src/plugins/discover/public/application/components/table/table_cell_value.tsx

```diff
@@ -7,60 +7,45 @@
  */
 
 import classNames from 'classnames';
-import React, { useCallback, useState } from 'react';
-import { IndexPatternField } from '../../../../../data/public';
+import React, { Fragment, useState } from 'react';
 import { FieldRecord } from './table';
 import { trimAngularSpan } from './table_helper';
 import { DocViewTableRowBtnCollapse } from './table_row_btn_collapse';
-import { DocViewTableRowIconUnderscore } from './table_row_icon_underscore';
 
 const COLLAPSE_LINE_LENGTH = 350;
 
-type TableFieldValueProps = FieldRecord['value'] & {
-  fieldName: string;
-  fieldMapping: IndexPatternField | undefined;
-};
+type TableFieldValueProps = FieldRecord['value'] & Pick<FieldRecord['field'], 'fieldKey'>;
 
-export const TableFieldValue = ({
-  formattedField,
-  fieldName,
-  fieldMapping,
-}: TableFieldValueProps) => {
+export const TableFieldValue = ({ formattedField, fieldKey }: TableFieldValueProps) => {
   const [fieldOpen, setFieldOpen] = useState(false);
 
-  const value = trimAngularSpan(String(formattedField));
+  const value = trimAngularSpan(String(formattedField || ''));
```

Comment:
would suggest keeping it the way before, can't think of a way formattedField could be anything else than a string, so even `String` should be redundant, but it's here because it can't be totally ruled out, and no imagine if formattedField would be: 0
anyway ... we soon should be able to remove the whole function

Reply:
Can It be undefined?

Reply:
I shouldn't be

Reply:
Let's keep it with a String wrapper  for now, like it was till this helper will be removed.

Reply:
ok with that